locales/base/translation.json

```diff
@@ -1664,6 +1664,10 @@
     "generateName": "Generate a name",
     "namePlaceholder": "Select a name ex) {{exampleName}}"
   },
+  "tokenBottomSheet": {
+    "searchAssets": "Search Assets",
+    "noTokenInResult": "No matching token for \"{{searchQuery}}\""
```

Comment:
I think this should be similar to the no text used in search so we can bold just the searchQuery / searchTerm to match the styling of Dapps Search: [Figma Comment](https://www.figma.com/file/jRtYI8GiSvPAaClBnRLLpu?node-id=2520:26583#363942229).
- [Translations file](https://github.com/valora-inc/wallet/pull/3599/files#diff-d3d67f4b3f8dbec345426f534268e40d9b087dfa5d83750165d63caf136cb24cR1288).
- [Styling in component](https://github.com/valora-inc/wallet/pull/3599/files#diff-0a07a8e746d1acf171a5b752ade521ea308d045d70283553323e2fc19af6d2cbR14-R27).